### DISCUSSION TITLE
✅ 🚀 Feature Request: Add 'exit to dashboard' to Top Bar experiment - [ED-10275]

### DISCUSSION THREAD

**Discussion by joehdevwp** [elementor/elementor#22359] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'm always frustrated when I'm using the Top Bar experiment and want to exit Elementor using my mouse to return to the dashboard.

### Describe the solution you'd like

Add (re-add) an 'exit to dashboard' button to the Elementor editor while Top Bar experiment is active. 

### Describe alternatives you've considered

Using the Finder. Doesn't always find results, requires typing

### Additional context

Am I the only one scratching his head how this could be missing? Or is it there and I am missing it??

**Comment by oscarhenao** [comment]
I do not understand how this can be left out. Great Top Bar missing a very important functionality!

**Comment by joehdevwp** [comment]
Looks like they added it back in v3.14. I take that as a silent admission that they made a mistake by removing it.  
Currently it doesn't respect the User Preference setting of exiting to All Posts, Dashboard, etc  but hey, we can't have everything. I mean, c'mon, Rome wasn't built in a day.

**Comment by nicholaszein** [comment]
Hi!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor  v3.14.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
